Makefile

```diff
@@ -27,4 +27,20 @@ all-tests:
 generate build clean:
 	go $@ $(EXTRAGOARGS)
 
+sandbox-test-fc-build:
+	docker build -f fctesting/sandbox/Dockerfile -t "firecracker" .
+	@touch sandbox-test-fc-build
+
+sandbox-test-fc-run:
+	docker run \
```

Comment:
Given the options here to grant pretty much all privileges and run as root, I'd like the make target to fail when invoked as a non-root user.  You should be able to do this by inserting `test "$(shell id --user)" -eq 0` above the `docker run` line.